bindings/java/iota-wallet-java/src/main/java/org/iota/Wallet.java

```diff
@@ -0,0 +1,400 @@
+        return new PreparedTransactionData(responsePayload);
+    }
+
+    public AccountBalance syncAccount(AccountIdentifier accountIdentifier, SyncAccount method) throws WalletException {
+        return GsonSingleton.getInstance().fromJson(callAccountMethod(accountIdentifier, method), AccountBalance.class);
+    }
+
+    public TransactionPayload sendAmount(AccountIdentifier accountIdentifier, SendAmount method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload sendMicroTransaction(AccountIdentifier accountIdentifier, SendMicroTransaction method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload sendNativeTokens(AccountIdentifier accountIdentifier, SendNativeTokens method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload sendNft(AccountIdentifier accountIdentifier, SendNft method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public void setAlias(AccountIdentifier accountIdentifier, SetAlias method) throws WalletException {
+        callAccountMethod(accountIdentifier, method);
+    }
+
+    public TransactionPayload sendOutputs(AccountIdentifier accountIdentifier, SendOutputs method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload signTransactionEssence(AccountIdentifier accountIdentifier, SignTransactionEssence method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload submitAndStoreTransaction(AccountIdentifier accountIdentifier, SubmitAndStoreTransaction method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public TransactionPayload claimOutputs(AccountIdentifier accountIdentifier, ClaimOutputs method) throws WalletException {
+        JsonObject responsePayload = (JsonObject) callAccountMethod(accountIdentifier, method);
+        return new TransactionPayload(responsePayload);
+    }
+
+    public void backup(String destination, String password) throws WalletException {
```

Comment:
You have some account manager methods at the top, then ` // Account Method APIs` and now again methods for the account manager

Reply:
The Message API is structured like this? :trollface:

Reply:
fixed